sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -1629,6 +1645,14 @@ aditof::Status CameraItof::parseJsonFileContent() {
             m_cameraFps = atoi(json_fps->valuestring);
         }
 
+        // Get fps from config
```

Comment:
The comment should say 'sync mode' instead of 'fps'

Reply:
missed that, thank you!